src/App.tsx

```diff
@@ -32,17 +58,25 @@ function App() {
       })
       .then(res => res.json())
       .then((data) => {
-      dispatch(login(data))
+          dispatch(login(data))
       })
+    // dispatch(getUser());
     }
+    // debugger;
+    // console.log(dispatch(fetchUser('hello')))
   },[])
- 
+
+  
+  
   return (
     <div className="App">
       <header>  
         <Navbar />
       </header>
-     
+      <div>
+        {localStorage.getItem('token') ?  <PublicChat /> : null}
```

Comment:
We should base this off a redux value without having to do another localstorage call here